test/clt-tests/migration-es-ms/init.recb

```diff
@@ -0,0 +1,15 @@
+––– input –––
+(dockerd > /var/log/dockerd.log 2>&1 &) > /dev/null
+––– output –––
+––– input –––
+if timeout 10 grep -qm1 'API listen on /var/run/docker.sock' <(tail -n 0 -f /var/log/dockerd.log); then echo 'Done'; else echo 'Timeout failed'; cat /var/log/dockerd.log; fi
+––– output –––
+Done
+––– input –––
+docker ps
+––– output –––
+CONTAINER ID   IMAGE     COMMAND   CREATED   STATUS    PORTS     NAMES
```

Comment:
I would like to expose this code to base block, cause it's the same as in other tests

Reply:
done